gravitee-node-api/src/main/java/io/gravitee/node/api/certificate/KeyStoreLoader.java

```diff
@@ -27,7 +27,7 @@ public interface KeyStoreLoader {
     String CERTIFICATE_FORMAT_PKCS12 = "PKCS12";
     String CERTIFICATE_FORMAT_SELF_SIGNED = "SELF-SIGNED";
     String CERTIFICATE_FORMAT_PEM_REGISTRY = "PEM-REGISTRY";
-    String GRAVITEEIO_PEM_REGISTRY = "graviteeio-pem-registry";
+    String GRAVITEEIO_PEM_REGISTRY_LABEL = "gravitee.io/component";
```

Comment:
This is not relates to keystore loader API
Can you move it to `io.gravitee.node.kubernetes.keystoreloader.KubernetesConfigMapKeyStoreLoader`